sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -254,7 +258,7 @@ es3fLifetimeTests.BufferVAOAttacher.prototype.detach = function(element, target)
 es3fLifetimeTests.BufferVAOAttacher.prototype.getAttachment = function(target) {
     var vao = /** @type {WebGLVertexArrayObject} */ (target);
     gl.bindVertexArray(vao);
-    var name = gl.getVertexAttrib(0, gl.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING);
+    var name = gl.getVertexAttrib(this.m_posLoc, gl.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING);
```

Comment:
See setPos, array buffer is bound to vertex attributes at index this.m_posLoc. So we should get buffer name from that index. I think C++ code has the same bug. In this case this.m_posLoc is always 0, so no difference between this.m_posLoc and 0.